pxe/common_files/disk_imaging.sh

```
@@ -930,6 +1134,8 @@ function main() {
 	fi
 }
 
-main
-echo "Rebooting" | tee $log_output
-systemctl reboot | tee $log_output
+if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
+	main
+	echo "Rebooting" | tee $log_output
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)